abci/types/types.pb.go

```diff
@@ -92,7 +92,7 @@ func (x ResponseOfferSnapshot_Result) String() string {
 }
 
 func (ResponseOfferSnapshot_Result) EnumDescriptor() ([]byte, []int) {
```

Comment:
we shouldn't be updating this file. have you synced with main branch? all abcix changes go to `abcix` folder